plugins/available/goenv.plugin.bash

```diff
@@ -0,0 +1,18 @@
+cite about-plugin
+about-plugin 'load goenv, if you are using it'
+
+# Don't modify the environment if we can't find the tool:
+# - Check if in $PATH already
+# - Check if installed manually to $HOME
+_command_exists goenv ||
+  [[ -f "$HOME/.goenv/bin/goenv" ]] ||
```

Comment:
In the context of the other notes in this review pass, even if this passes, it may not make it to the path.

Even though my notes say to not set GOENV_ROOT to any default values (per your previous comments on not overriding the tool's builtin defaults), you *might* consider setting GOENV_ROOT to `~/.goenv` when:
1) `goenv` is not already on the path
2) `GOENV_ROOT` is not already set
3) `~/.goenv/bin/goenv` exists

Reply:
nope. i don't know what isn't clear about we want to be explicit when the user has gone through the work to activate a plugin.